schema-guide.md

```diff
@@ -1912,7 +1912,7 @@ authors:
 
 ### `definitions.orcid`
 
-- **type**: `uri` with pattern [`https://orcid\.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]{1}`](https://regex101.com/library/wvvVYE)
+- **type**: `uri` with pattern [`^https://orcid\.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]{1}$`](https://regex101.com/r/GLGGDO/1).
```

Comment:
I wonder whether the difference in the URL routes matters

Reply:
Wondered about this in the past, but there's no info on the Wiki. I don't think it's critical just now for the link to always work. The pattern is the more important part... I think saving it in a library may give you more control...